### PR TITLE
Fix flakey tests

### PR DESCRIPTION
**Pull request by dalebennett1992**

### CODE REVIEW

File: .github/scripts/poll_for_endpoint_health.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import fetch from 'node-fetch';
+import smokeConsoleTestnet from '../../config/console-smoke-testnet.js';
+import smokeConsoleMainnet from '../../config/console-smoke-mainnet.js';
+
+const networkArg = process.argv[2];
+
+if (!networkArg || (networkArg !== 'testnet' && networkArg !== 'mainnet')) {
+  console.error('Usage: node script.js [testnet/mainnet]');
+  process.exit(1);
+}
```
Comment (dexturr) [inline .github/scripts/poll_for_endpoint_health.js:10]:
```suggestion
const PERMITTED_NETWORKS = ['testnet', 'mainnet']

if (!PERMITTED_NETWORKS.inckudes(networkArg)) {
  console.error(`Usage: network must be provided and must be one of ${PERMITTED_NETWORKS.join(' ')}`);
  process.exit(1);
}
```

File: .github/scripts/poll_for_endpoint_health.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+const endpointUrl = networkArg === 'testnet' ? smokeConsoleTestnet.network.console : smokeConsoleMainnet.network.console;
```
Comment (dexturr) [inline .github/scripts/poll_for_endpoint_health.js:12]: Possibly makes more sense, though is async so adds some complexity. Will let you make a call

```suggestion
const endpointUrl = await import(path + networkArg).default.network.console
```

**Review by dexturr — COMMENTED** [review]
Couple minor comments

Just a thought as well that this can still flake if fairground is down. Perhaps this script should attempt to check that and error with an appropriate message if that is the case, so that we can keep an eye on if it's happening a lot and deal with appropriately if required.

File: .github/scripts/poll_for_endpoint_health.js (hunk 2 above)
Comment (dalebennett1992) [reply .github/scripts/poll_for_endpoint_health.js:12]: I think it is simple enough for now to leave, if we was supporting more networks I would change

**Review by dexturr — APPROVED** [review]

**Pull request merged**